src/components/Calculator.js

```diff
@@ -0,0 +1,38 @@
+const Calculator = () => (
+  <div className="calculator">
+    <input type="text" className="display-result" id="result" value="0" readOnly />
+    <div className="functional-buttons">
+      <button type="button" id="ac">AC</button>
+      <button type="button" id="+/-">+/-</button>
+      <button type="button" id="%">%</button>
+      <button type="button" className="orange-btn" id="/">
+        /
+      </button>
+      <button type="button" id="7">7</button>
+      <button type="button" id="8">8</button>
+      <button type="button" id="9">9</button>
+      <button type="button" className="orange-btn" id="*">
+        *
+      </button>
```

Comment:
- [ ] It would be better if our Calculator components has 1 child component, for example, we create create a new file and call it `Button` component and import the Button component in the Calculator component as the child component like below
```
import Button from './button.js';
src/components/Calculator.js
const Calculator = () => (
   <input type="text" className="display-result" id="result" value="0" readOnly />
   <Button />
);
export default Calculator;

```
![Screenshot from 2023-06-05 17-37-51](https://github.com/DebyGrey/math-magicians/assets/46388113/9f3f9c4b-9b78-4c0f-ae7e-a93a88ee245c)